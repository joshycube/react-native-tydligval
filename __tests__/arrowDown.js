import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import ArrowDown from "../svg/arrowDown";

describe("arrowDown", () => {
  test("renders the SVG correctly", () => {
    const shallow = new ShallowRenderer();
    shallow.render(<ArrowDown />);
    const result = shallow.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
