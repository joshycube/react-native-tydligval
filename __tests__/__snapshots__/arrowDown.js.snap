// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`arrowDown renders the SVG correctly 1`] = `
<Svg
  height="10"
  preserveAspectRatio="xMidYMid meet"
  width="10"
>
  <G
    fill="#ffffff"
    fillRule="evenodd"
    stroke="none"
    strokeWidth="1"
  >
    <G
      fill="#ffffff"
      transform="translate(-997.000000, -572.000000)"
    >
      <G
        transform="translate(997.000000, 572.000000)"
      >
        <Polygon
          points="0 0 5.21196348 6.21196348 10.9042969 0"
        />
      </G>
    </G>
  </G>
</Svg>
`;
